src/runtimes/docker/contracts/ContainerClient.ts

```diff
@@ -145,6 +146,10 @@ export type InfoItem = {
      * The OS type for the container runtime
      */
     osType?: ContainerOS;
+    /**
+     * Docker Client info record
+     */
+    clientInfo?: DockerClientInfo;
```

Comment:
Define a ClientInfo type in this file rather than importing the type from DockerInfoRecord.

@bwateratmsft we should probably add a work item to implement a "feature support" check like we've discussed in the past. It would let us cleanup this and several other cases where we have to leak Docker implementation details into the extension UI logic.